app/models/host_validator.rb

```diff
@@ -1,6 +1,6 @@
 class HostValidator
-  INVALID_HOSTS = %w(127.0.0.1 localhost)
-  INVALID_TLDS = %w(.dev)
+  INVALID_HOSTS = %w(127.0.0.1 localhost example.com example.org example.net example.dev)
```

Comment:
This sent me down a bit of a rabbithole - thank you!

I first visited https://example.com, which led me to https://www.iana.org/domains/reserved, which led me to [RFC 2606](https://tools.ietf.org/html/rfc2606) and [RFC 6761](https://tools.ietf.org/html/rfc6761), which describe example-type domains.

RFC 2606 specifically mentions:

* https://example.com
* https://example.net
* https://example.org

as reserved, all of which do indeed lead to IANA-maintained pages. 

It looks like `whois example.dev` shows that it's maintained by Google, `whois example.blog` is run by Automattic's registrar ("Knock Knock WHOIS there" 😄), and so it seems that indeed there's a pattern of `example.TLD` domains being registered.

Given that, what do you think about banning all `example.TLD` domains (using a method, not a giant blacklist array)? That's probably best done in a separate PR, so I suggest:

* leaving the `INVALID_HOSTS` variable alone in this PR
* opening a new PR (or I can do it) to ban all hosts with a domain of `example`.

Reply:
> what do you think about banning all example.TLD domains (using a method, not a giant blacklist array)? That's probably best done in a separate PR

That sounds good to me! I'm happy to open that PR tomorrow morning.

Reply:
Thanks!